larray/core.py

```diff
@@ -9139,14 +9145,48 @@ def from_string(s, nb_index=None, index_col=None, sep=','):
         List of columns for the index (ex. [0, 1, 2, 3]). Defaults to None (see nb_index above).
     sep : str
         delimiter used to split each line into cells.
+    \**kwargs
+
+        * parse_dates : boolean or list of ints or names or list of lists or dict, default False
+
+            * boolean. If True -> try parsing the index.
+            * list of ints or names. e.g. If [1, 2, 3] -> try parsing columns 1, 2, 3
+              each as a separate date column.
+            * list of lists. e.g.  If [[1, 3]] -> combine columns 1 and 3 and parse as a single date column.
+            * dict, e.g. {'foo' : [1, 3]} -> parse columns 1, 3 as date and call result 'foo'
+
+        Note: A fast-path exists for iso8601-formatted dates.
+
+        * infer_datetime_format : boolean, default False
+
+            If True and parse_dates is enabled, pandas will attempt to infer the format
+            of the datetime strings in the columns, and if it can be inferred, switch
+            to a faster method of parsing them. In some cases this can increase the
+            parsing speed by ~5-10x.
+
+        * keep_date_col : boolean, default False
+
+            If True and parse_dates specifies combining multiple columns then keep the original columns.
+
+        * date_parser : function, default None
+
+            Function to use for converting a sequence of string columns to an array of
+            datetime instances. The default uses ``dateutil.parser.parser`` to do the
+            conversion. Pandas will try to call date_parser in three different ways,
+            advancing to the next if an exception occurs: 1) Pass one or more arrays
+            (as defined by parse_dates) as arguments; 2) concatenate (row-wise) the
+            string values from the columns defined by parse_dates into a single array
+            and pass that; and 3) call date_parser once for each row using one or more
+            strings (corresponding to the columns defined by parse_dates) as arguments.
+
+        See other arguments of pandas read_csv method.
```

Comment:
generally, I don't like this way of documenting kwargs. I would rather have them either as explicit kwargs in the function signature OR not document them and say that all extra kwargs are forwarded to pandas read_csv and invite people to look at that.